src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
```

Comment:
The `health` property should be initialized in the constructor to ensure each instance of `Animal` has its own health value. Move `health = 100;` inside the constructor.